.github/workflows/check-license.yml

```
@@ -2,19 +2,20 @@ name: License Header Check
 
 on:
   pull_request:
-    branches:
-      - develop
   push:
     branches:
       - develop
 
+permissions:
+  contents: read
+
 jobs:
   check-license-header: 
     name: Check License Header
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v6
+      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3
```

```suggestion
      - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8
```

I'm seeing `8e8c483db84b4bee98b60c0593521ed34d9990e8` for `v6`

Ref: https://github.com/actions/checkout/releases/tag/v6